apps/crawler/src/scanners/accounts-scanner.service.ts

```diff
@@ -0,0 +1,112 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { Repository } from 'typeorm';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Account } from '@entities/Account';
+import { SdkService } from '../skd/sdk.service';
+import { normalizeSubstrateAddress, normalizeTimestamp } from '@common/utils';
+import { AllBalances } from '@unique-nft/sdk/types';
+import IScannerService from './scanner.interface';
+import { ConfigService } from '@nestjs/config';
+
+@Injectable()
+export class AccountsScannerService implements IScannerService {
+  private readonly logger = new Logger(AccountsScannerService.name);
+
+  constructor(
+    @InjectRepository(Account)
+    private accountsRepository: Repository<Account>,
+    private configService: ConfigService,
+    private sdkService: SdkService,
+  ) {}
+
+  async scan() {
+    this.logger.log('Start full scan...');
+
+    const PARALLEL_TASKS = this.configService.get('PARALLEL_TASKS') | 10;
```

Comment:
Typo? Mabe you mean || "logic or" instead of "bit or".

Reply:
I felt that strange feeling that something's wrong...) Fixed.